TRAVIS

```diff
@@ -9,34 +16,31 @@ Implement a track test suite that can run both locally and on Travis CI. The tra
 
-- helps triangulate a solution to be more generic, or
-- requires new functionality incrementally.
+_Enabling all test cases_
 
-Many test frameworks will randomize the order of the tests when running them. This is an excellent practice, which helps ensure that subsequent tests are not dependent on side effects from earlier tests. However, in order to simulate TDD we want tests to run *in the order that they are defined*, and we want them to *fail fast*, that is to say, as soon as the test suite encounters a failure, we want the execution to stop. This ensures that the person implementing the solution sees only one error or failure message at a time, unless they make a change which causes prior tests to fail.
+As described in [Contributing to Exercism: Writing exercise test suites](writing_tests.md), exercise test suites are written such that all but the first test case are skipped/pending/ignored.  For the purpose of the track test suite, all test cases should be enabled (simulating an Exercism user having completed the exercise).
 
-This is the same experience that they would get if they were implementing each new test themselves.
+Additionally, in some programming languages, the name of the file containing the solution is hard-coded in the test suite, and the example solution is not named in the way that we expect people to name their files.
 
-Most testing frameworks do not have the necessary configuration options to get this behavior directly, but they often do have a way of marking tests as _skipped_ or _pending_. The mechanism for this will vary from language to language and from test framework to test framework.
+We will need to temporarily (and programmatically) edit the exercise test suites to ensure that all of their tests are active. We may also need to rename the example solution file(s) in order for the exercise test suite to run against it.
 
-Whatever the mechanism—functions, methods, annotations, directives, commenting out tests, or some other approach—these are changes made directly to the test file. The person solving the exercise will need to edit the test file in order to "activate" each subsequent test.
+It's important that if we rewrite files in any way during a test run, that these changes do not accidentally get checked in to the git repository.
 
-Any tests that are marked as skipped will not be verified by the track test suite unless special care is taken.
+Therefore, many language tracks write the track test suite in such a way that it _copies_ the exercise to a temporary location outside of the git repository before editing or rewriting the exercise files during a test run.
 
-Additionally, in some programming languages, the name of the file containing the solution is hard-coded in the test suite, and the example solution is not named in the way that we expect people to name their files.
+_Configuring Travis CI_
```

Comment:
I now realize that we sometimes have AppVeyor (Microsoft stack) and Circle CI (Apple stack). Travis CI only runs on Linux.